lib/index.js

```diff
@@ -238,9 +238,19 @@ exports.notFound = function (message, data) {
 };
 
 
-exports.methodNotAllowed = function (message, data) {
+exports.methodNotAllowed = function (message, data, allow) {
 
-    return internals.create(405, message, data, exports.methodNotAllowed);
+    const err = internals.create(405, message, data, exports.methodNotAllowed);
+
+    if (typeof allow === 'string') {
+        allow = [allow];
+    }
+
+    if (Array.isArray(allow)) {
```

Comment:
Happy to make it clearer if needed, but that's correct, it would break if it were an if/else.